core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -160,6 +160,7 @@ def _fit(self, X_train, y_train, k_fold=5, k_fold_start=0, k_fold_end=None, n_re
             self._add_child_times_to_bag(model=model_base)
             return
 
+        sample_weights = kwargs.get('sample_weights', None)
```

Comment:
For clarity, explicitly have `sample_weight` as an argument. This is to show what optional functionality each model uses in fit, and kwargs is generally ignored. This applies to all models that use `sample_weight` and `sample_weight_val`.